src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -260,8 +261,8 @@ export default class AmalgamatingBusinesses extends Mixins(AmalgamationMixin, Co
     this.jurisdiction = jurisdiction
     this.isCan = jurisdiction.group === 0
     this.jurisdictionErrorMessage = this.jurisdiction ? '' : 'Home jurisdiction is required'
-    this.isMrasJurisdiction = ['AB', 'MB', 'NS', 'ON', 'QC', 'SK'].includes(
-      this.jurisdiction.value
+    this.isMrasJurisdiction = MrasJurisdictions.includes(
+      this.jurisdiction.text.toLowerCase()
```

Comment:
I think `MrasJurisdictions` is full words while `jurisdiction` is an abbrev?

Reply:
`jurisdiction` contains both abbreviations (`jurisdiction.value`) and the full words (`jurisdiction.text`)

Reply:
Perfect. Yes, I missed the `.text` in your code above. Resolving!